src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala

```diff
@@ -33,24 +32,16 @@ class ImapFullScenario {
   val lightUser = receiveEmail
     .exec(readLastEmail)
 
-  def generate(duration: Duration, users: Seq[Future[User]]): ScenarioBuilder =
+  def generate(duration: Duration, feeder: FeederBuilder): ScenarioBuilder =
     scenario("Imap")
-      .feed(UserFeeder.createCompletedUserFeederWithInboxAndOutbox(users))
+      .feed(feeder)
       .pause(1.second)
+      .exec(imap("Connect").connect()).exitHereIfFailed
+      .exec(imap("login").login("${username}", "${password}").check(ok))
       .during(duration) {
```

Comment:
you changed the semantic of this scenario:
* before: choose a profile and during duration, do repeatedly this profile
* after: during duration, repeatedly choose a profile and execute it
Am I right ?

Reply:
Yes, and i think neither of these solutions are really good.
So i created another issue :
https://ci.linagora.com/linagora/lgs/openpaas/james/issues/1190

But tell me if i didn't understood correctly the scenario, if its the case i will revert the commit on this as it doesn't concern this PR in the first place.